### ISSUE TITLE
{handshake_timeout,frame_header}

### ISSUE THREAD

**Issue by EagleEyeJohn** [pdezwart/php-amqp#108]
Are you able to shed some light on why we should be receiving the following log entries when using version 1.4.0-1?  Attempts to connect fail with ```{handshake_timeout,frame_header}```.

These are intermittent and do not appear to relate to high load on the server which has a load average of 0.0x most of the time.  The publish rate to RabbitMQ is less than 0.5/sec.

```
=INFO REPORT==== 27-Apr-2014::05:05:02 ===
accepting AMQP connection <0.22398.229> (127.0.0.1:36025 -> 127.0.0.1:5672)

=INFO REPORT==== 27-Apr-2014::05:05:02 ===
accepting AMQP connection <0.22401.229> (127.0.0.1:36026 -> 127.0.0.1:5672)

=ERROR REPORT==== 27-Apr-2014::05:05:12 ===
closing AMQP connection <0.22398.229> (127.0.0.1:36025 -> 127.0.0.1:5672):
{handshake_timeout,frame_header}

=ERROR REPORT==== 27-Apr-2014::05:05:12 ===
closing AMQP connection <0.22401.229> (127.0.0.1:36026 -> 127.0.0.1:5672):
{handshake_timeout,frame_header}
```

```
# php -v
PHP 5.5.10 (cli) (built: Mar  6 2014 10:44:40)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
```

```
# php --ri amqp

amqp

Version => 1.4.0
Revision => $Revision: 327551 $
Compiled => Apr 23 2014 @ 09:03:55
AMQP protocol version => 0-9-1
librabbitmq version => 0.5.0

Directive => Local Value => Master Value
amqp.host => localhost => localhost
amqp.vhost => / => /
amqp.port => 5672 => 5672
amqp.timeout => no value => no value
amqp.read_timeout => 0 => 0
amqp.write_timeout => 0 => 0
amqp.connect_timeout => 0 => 0
amqp.login => guest => guest
amqp.password => guest => guest
amqp.auto_ack => 0 => 0
amqp.prefetch_count => 3 => 3
```

```
# yum list installed | grep rabbit
librabbitmq.x86_64               0.5.0-1.el6.remi          @remi
librabbitmq-devel.x86_64         0.5.0-1.el6.remi          @remi
librabbitmq-tools.x86_64         0.5.0-1.el6.remi          @remi
rabbitmq-server.noarch           3.3.0-1                   installed
```

```
# yum list installed | grep php
php55u.x86_64                    5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-cli.x86_64                5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-common.x86_64             5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-devel.x86_64              5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-gd.x86_64                 5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-gmp.x86_64                5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-imap.x86_64               5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-mbstring.x86_64           5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-mysqlnd.x86_64            5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-pdo.x86_64                5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-pear.noarch               1:1.9.4-20.ius.el6        @rackspace-rhel-x86_64-server-6-ius
php55u-pecl-jsonc.x86_64         1.3.5-1.ius.el6           @rackspace-rhel-x86_64-server-6-ius
php55u-process.x86_64            5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
php55u-xml.x86_64                5.5.10-1.ius.el6          @rackspace-rhel-x86_64-server-6-ius
```

```
# cat /etc/redhat-release
Red Hat Enterprise Linux Server release 6.5 (Santiago)
```

**Comment by lstrojny** [comment]
@EagleEyeJohn still having the issue?

**Comment by lstrojny** [comment]
Closing

**Comment by pinepain** [comment]
This issue should be fixed in 1.6.0.
